crates/sui-rpc-benchmark/src/bin/process_jsonl.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// This script processes JSONL logs of timestamps, hosts, methods and bodies.
+/// It filters specific RPC calls within a sliding time window and writes
+/// aggregated results (with iteration data) to an output JSONL file.
+/// This is useful for methods of suix_getOwnedObjects, suix_getAllCoins, suix_getCoins,
+/// b/c the `cursor` has changed to encode more information, thus old cursors are not valid
+/// on the `alt` stack.
+///
+/// Usage Example:
+///     cargo run --bin process_jsonl -- \
+///       --input /path/to/input.jsonl \
+///       --output /path/to/output.jsonl \
+///       --window_minutes 60
+use chrono::{DateTime, Duration, Utc};
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::collections::HashMap;
+use std::error::Error;
+use std::fs::File;
+use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::path::PathBuf;
+use tracing::{debug, info};
+
+const METHODS_TO_AGGREGATE: [&str; 3] =
+    ["suix_getOwnedObjects", "suix_getAllCoins", "suix_getCoins"];
+const WALLET_HOST_PREFIX: &str = "wallet";
+
+#[derive(Debug, Serialize, Deserialize)]
+struct LogEntry {
```

Comment:
this struct is tech. duplicate with pull_grafana_logs, leaving it here so that
- no extra dependencies for simplicity
- the dup is minimal and I do not expect many incremental changes looking forward so it not extra burden to carry